src/dbus/server/introspect.xml

```diff
@@ -246,8 +246,15 @@
       <arg name="epskc" type="s" direction="out"/>
     </method>
 
-    <!-- DeactivateEphemeralKeyMode: Deactivate ePSKc mode. -->
+    <!-- DeactivateEphemeralKeyMode: Deactivate ePSKc mode.
+      @retain_active_session: 
+        "false" - Disconnects the Border Agent from any active secure sessions.
+                  Ephemeral key would be cleared after the session is disconnected.
+        "true" - Deactivate ephemeral key mode softly. If a commissioner connection is active,
+                 wait for it to disconnect or for the ephemeral key timeout.
```

Comment:
Sorry, I think the second sentence is also not accurate. I would rephrase as:

```
"true" - Deactivate ephemeral key mode softly. If there is already an active commissioner connection, the session will be retained
```

Reply:
Changed accordingly to reduce misleading, thanks!